### PR TITLE
chore: backmerge develop into v5 main

### PR DESCRIPTION
**Pull request by Marc-Roig**
Backmerge

### CODE REVIEW

**Comment by Marc-Roig** [comment]
we are going to validate with nathan the cloud changes work ok

**Review by alexandrebodin — APPROVED** [review]

File: packages/generators/app/src/resources/files/js/README.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ yarn build
 
 Strapi gives you many possible deployment options for your project including [Strapi Cloud](https://cloud.strapi.io). Browse the [deployment section of the documentation](https://docs.strapi.io/dev-docs/deployment) to find the best solution for your use case.
 
+```
+yarn strapi deploy
+```
+
```
Comment (nathan-pichon) [inline packages/generators/app/src/resources/files/js/README.md:42]:
```suggestion
[comment]: <> (TODO - To be included when Cloud is v5 compatible)
[comment]: <> (```)
[comment]: <> (yarn strapi deploy)
[comment]: <> (```)
```

https://stackoverflow.com/a/20885980/12444940

File: packages/generators/app/src/resources/files/ts/README.md [hunk 2]
```diff
@@ -36,6 +36,10 @@ yarn build
 
 Strapi gives you many possible deployment options for your project including [Strapi Cloud](https://cloud.strapi.io). Browse the [deployment section of the documentation](https://docs.strapi.io/dev-docs/deployment) to find the best solution for your use case.
 
+```
+yarn strapi deploy
+```
+
```
Comment (nathan-pichon) [inline packages/generators/app/src/resources/files/ts/README.md:42]:
```suggestion
[comment]: <> (TODO - To be included when Cloud is v5 compatible)
[comment]: <> (```)
[comment]: <> (yarn strapi deploy)
[comment]: <> (```)
```

**Review by nathan-pichon — APPROVED** [review]

**Comment by Marc-Roig** [comment]
thank you @nathan-pichon for all the help <3

**Pull request merged**